### ISSUE TITLE
Gulp-Man (1994)

### ISSUE THREAD

**Issue by bigjim161** [exoscoriae/eXoDOS#1781]
This games is not on the "Dos masters list"and on mobygames.
screenshots: https://www.old-games.ru/game/screenshots/10112.html

Genre: Arcade
Developer: Carlos Esteve Cremades
Publisher: Ediciones Mago
Released: 1994

Added to the github folder

**Comment by TimberlAndRE-de** [comment]
@bigjim161 I think this one is spanish only, maybe something for the future spanish package

**Comment by bigjim161** [comment]
Yes it's Spanish, but in the game genre, the language doesn't seem to be a problem to play.

**Comment by TimberlAndRE-de** [comment]
Ah, ok 👍🏻
